app/src/main/res/values-fr/strings.xml

```diff
@@ -314,11 +314,11 @@
     <string name="always_default_web_search_on_enter">Toujours utiliser la recherche web par défaut lors d\'une saisie de texte</string>
     <string name="results_size">Taille d\'affichage des résultats</string>
     <string name="large_result_list_margins">Marges des listes plus grandes</string>
-    <string name="reset_excluded_app_shortcuts_name">Réinitialiser la liste des applications exclues de l\'affichage des raccoucis</string>
+    <string name="reset_excluded_app_shortcuts_name">Réinitialiser la liste des applications exclues de l\'affichage des raccourcis</string>
     <string name="reset_excluded_app_shortcuts_warn">Voulez-vous vraiment réinitialiser votre liste d\'applications exclues de l\'affichage des raccourcis \?</string>
     <string name="ui_excluded_from_shortcuts_apps">Voir ou éditer les applications exclues de l\'affichage des raccourcis</string>
     <string name="color_system">Couleurs du système</string>
-    <string name="icons_themed_summary">Les icônes sont teinté avec les couleurs du système (bêta)</string>
+    <string name="icons_themed_summary">Les icônes sont teintées avec les couleurs du système (bêta)</string>
```

Comment:
Missing plural, plus icône even though there is debate on whether "icône" should be masculine or feminine, it _is_ feminine everywhere else in this file.